hooks/build

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+rm -rf bin Player composer.* .php-cs-fixer.dist.php box.* Dockerfile-dev Makefile phive.xml phpstan.neon phpunit.xml.dist
```

Comment:
Not sure if you need to remove them - as you are not adding any files from repository into your docker file - it should not affect image size at all.

Reply:
:+1: 

@lolautruche if you want to avoid sending files to the Docker daemon (for performance) when building the image, use the `.dockerignore` file for that (See https://docs.docker.com/engine/reference/builder/#dockerignore-file)

Reply:
That's indeed a very good point :)